config/locales/fr.yml

```diff
@@ -0,0 +1,56 @@
+fr:
+  money:
+    format: '%n %u'
+    decimal_mark: ','
+    thousands_separator: ' '
+  date:
+    abbr_day_names:
+      - Dim
+      - Lun
+      - Mar
+      - Mer
+      - Jeu
+      - Ven
+      - Sam
+    abbr_month_names:
+      -
+      - Jan
+      - Fév
+      - Mar
+      - Avr
+      - Mai
+      - Juin
+      - Jui
+      - Aoû
+      - Sep
+      - Oct
+      - Nov
+      - Déc
+    day_names:
+      - Dimanche
+      - Lundi
+      - Mardi
+      - Mercredi
+      - Jeudi
+      - Vendredi
+      - Samedi
+    formats:
+      default: '%d %b %Y'
+    month_names:
+      -
```

Comment:
Why this line? 🤔

Reply:
Not sure tbh, but has to be there. E.g. https://github.com/svenfuchs/rails-i18n/blob/master/rails/locale/en.yml

Reply:
fair enough 😄